guides-and-standards/standards--source-control.md

```diff
@@ -0,0 +1,153 @@
+# Standards for source control
+
+This document describes the software engineering standards related to version control at Arcadia Science.
+
+For a TLDR overview, see the [core standards](#core-standards) section below.
+
+## Table of contents
+
+- [Standards for source control](#standards-for-source-control)
+  - [Table of contents](#table-of-contents)
+  - [About Git and GitHub](#about-git-and-github)
+  - [Core standards](#core-standards)
+  - [Repositories](#repositories)
+    - [Public repositories](#public-repositories)
+    - [Repository templates](#repository-templates)
+    - [Default repository settings](#default-repository-settings)
+    - [\[Optional\] Code owners](#optional-code-owners)
+  - [Branches and pull requests](#branches-and-pull-requests)
+    - [Working on branches](#working-on-branches)
+    - [Pull Requests (PRs) and code reviews](#pull-requests-prs-and-code-reviews)
+    - [What not to keep under version control](#what-not-to-keep-under-version-control)
+    - [Review templates](#review-templates)
+    - [\[Optional\] Reviewing Jupyter notebooks using the ReviewNB tool](#optional-reviewing-jupyter-notebooks-using-the-reviewnb-tool)
+  - [Data management](#data-management)
+  - [Guidelines for collaborating on codebases with collaborators who are not Arcadians](#guidelines-for-collaborating-on-codebases-with-collaborators-who-are-not-arcadians)
+  - [Common pitfalls and tricks to overcome them](#common-pitfalls-and-tricks-to-overcome-them)
+  - [Tutorials](#tutorials)
+  - [Credits](#credits)
+
+## About Git and GitHub
+
+Version control is the practice of tracking and managing changes to software code. This means we can easily see who made what change when and potentially revert to earlier version of the code. It is an essential collaboration tool for large teams to build complex software. [Git](https://git-scm.com/) is a free and open source distributed version control system. Since version control tools like Git work across platforms, it also makes access to code repositories easy and consistent across environments.
+
+[GitHub](https://github.com/) is built on top of Git and is our primary version control service. We have an [Arcadia Science account](https://github.com/Arcadia-Science) and we expect that team members will maintain their code in repositories under this team account.
```

Comment:
May be worth adding that ppl can use their existing github username, they don't need to create a new account for Arcadia. maybe a little redundant bc most people looking at this will have already "signed up" for github, so no worries if you don't want to add that

Reply:
Added. This repo is (somewhat surprisingly) public, so readers won't necessarily have signed up or been added to the arcadia github org.